docs/manage-your-account/workspace-usage.mdx

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 3
+title: Workspace Usage
+description: "Understand your workspace usage"
+---
+
+import orgUsage from "/static/ui-videos/usage/usage-fhd.mp4";
+
+Keeping track of your workspace usage is essential to ensure that your team is operating at peak efficiency and that your
+organization is getting the most out of Permit.
+
+## Workspace Usage
+
+To view your workspace usage, go to the **Projects** screen on the Sidebar.
+The workspace usage is displayed in the top right corner of the screen.
+In addition to the current usage, clicking on the (?) icon will show you the usage history of your workspace tenants and MAUs.
+
+<video controls autoPlay loop muted playsInline>
+  <source src={orgUsage} type="video/mp4" />
+</video>
+
+The **Workspace Usage** screen gives you a comprehensive overview of your workspace's usage. It shows you the number of:
```

Comment:
I think there is no need to mention "Workspace usage" so many times. Let's just say:
This section gives you a comprehensive overview of your current usage. It shows you the number of:"

Reply:
Done, also changed the title